GameServerCore/Packets/Interfaces/IPacketNotifier.cs

```diff
@@ -544,7 +544,8 @@ public interface IPacketNotifier
         /// </summary>
         /// <param name="clientInfo">Information about the client which had their hero created.</param>
         /// <param name="userId">User to send the packet to. Set to -1 to broadcast.</param>
-        void NotifyS2C_CreateHero(ClientInfo clientInfo, int userId = -1);
+        /// <param name="doVision">Whether or not to package the packets into a vision packet.</param>
```

Comment:
Although this option can be streamlined for all other packets, I'll accept for now.